R/selections.R

```diff
@@ -204,6 +204,19 @@ all_predictors <- function() {
   has_role("predictor")
 }
 
+#' @export
+#' @rdname has_role
+all_numeric_predictors <- function() {
```

Comment:
Add some docs for these two in `?has_role`?

Reply:
Thats the topic that has all of the selector information.

Reply:
I changed the docs to mention them more in ways that the user will see.